pkg/backup/util/util.go

```diff
@@ -86,24 +86,40 @@ func GenerateS3CertEnvVar(s3 *v1alpha1.S3StorageProvider) ([]corev1.EnvVar, stri
 			Name:  "AWS_STORAGE_CLASS",
 			Value: s3.StorageClass,
 		},
-		{
-			Name: "AWS_ACCESS_KEY_ID",
-			ValueFrom: &corev1.EnvVarSource{
-				SecretKeyRef: &corev1.SecretKeySelector{
-					LocalObjectReference: corev1.LocalObjectReference{Name: s3.SecretName},
-					Key:                  constants.S3AccessKey,
+	}
+	accessKeyName := "AWS_ACCESS_KEY_ID"
+	secretKeyName := "AWS_SECRET_ACCESS_KEY"
+	if useKMS {
+		envVars = append(envVars, []corev1.EnvVar{
+			{
+				Name:  "AWS_DEFAULT_REGION",
+				Value: s3.Region,
```

Comment:
What if region is not specified? The script in entrypoint checks the environment variable `AWS_DEFAULT_REGION`.

Reply:
the  AWS_DEFAULT_REGION have to set If we use KMS to decrytption. see https://github.com/pingcap/dbaas/pull/962  decription step 5